app/src/main/java/org/matrix/matrixandroidsdk/services/EventStreamService.java

```diff
@@ -264,14 +266,20 @@ private void startWithNotification() {
         mState = StreamAction.START;
     }
 
-    private Notification buildMessageNotification(String from, String body, String roomId) {
+    private Notification buildMessageNotification(String from, String body, String roomId, String roomName) {
         NotificationCompat.Builder builder = new NotificationCompat.Builder(this);
         builder.setWhen(System.currentTimeMillis());
         builder.setContentTitle(from);
+        String name = null;
+        if("" == roomName || null == roomName) {
```

Comment:
You can use ``TextUtils.isEmpty(roomName)`` here from ``android.text``, which conveys your intent more clearly.